web-server/controllers/manager-controller.js

```diff
@@ -316,6 +330,36 @@ class ManagerController extends UserController {
             req.whoops();
         }
     }
+
+
+    /**
+     * Gets a prosumers latest production data.
+     */
+    async getCurrentProductionData(req, res) {
+        try {
+            const response = await fetch(`http://simulator:3000/simulator/prosumer/${req.body.prosumerId}`);
```

Comment:
I think you could just have used axios, now we have two libraries that does the same thing.